actions/poll.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  ConditionalOrderFactory,
+  DEFAULT_CONDITIONAL_ORDER_REGSTRY,
+  PollResult,
+  PollResultCode,
+} from "@cowprotocol/cow-sdk";
+
+import { PollingParams } from "./model";
+
+const ordersFactory = new ConditionalOrderFactory(
+  DEFAULT_CONDITIONAL_ORDER_REGSTRY
+);
+
+export async function pollConditionalOrder(
+  params: PollingParams
+): Promise<PollResult | undefined> {
+  const { owner, chainId, conditionalOrderParams, provider } = params;
+
+  const order = ordersFactory.fromParams(params.conditionalOrderParams);
```

Comment:
now we just pass blindly the params, and the factory will create the `ConditionalOrder` for us

After that we can use the SDK methods from the order, including of course the polling.